pages/badges/badges.jsx

```diff
@@ -155,6 +155,7 @@ var BadgesPage = React.createClass({
   },
 
   generateBadgeList: function() {
+    var that = this;
```

Comment:
Hmm, this should not be necessary. If we need to refer to the instance, then the function later on should love on the React component and passed into the handler as `this.[functionname]`, rather than aliassing 'that' (since React does autorebinding for component functions)